src/gwt/CMakeLists.txt

```diff
@@ -60,7 +80,13 @@ if(GWT_BUILD)
    endif()
 
    # invoke ant to build
-   add_custom_target(gwt_build ALL)
+   if(GWT_BUILD_ALL)
```

Comment:
In a dev workflow, we don't build GWT (typically) as part of the overall build. Instead the developer will go into `rstudio/src/gwt` and run `ant devmode`. This runs a recompile-on-demand service that is much faster than a full production build (`ant`) and supports rebuilding when you refresh the browser showing the IDE.

Pro adds some other wrinkles, but let's tackle that after dealing with open-source

Reply:
So, if we're wanting to put the recommended dev workflow into the container, we don't want the build to include GWT. There are existing ways to tell cmake to skip it (can chat about that offline if not obvious).

Then our "how to run the server, and iterate on the code" instructions will include the step of running `ant devmode`.

This will probably eliminate the need for messing with the Java resource switches, though having that capability is worth having just in case.

Reply:
In this PR, we set the `GWT_BUILD_ALL` var for CMake to `NO`, so we do exclude it from the `ALL` target by default. Is this ok, or are you suggesting that we accomplish it another way?

Reply:
I removed this logic since #9885 modified the GWT build to run in isolation.